.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,17 @@
-<!-- Complete this Pull Request template. -->
+<!--
+Thanks for your contribution:
+* Please list any related issues with a "closes" or "addresses" tag.
+* For bugfixes we have maintenance branches e.g. `8.0.x`, please raise separate
+  pull requests against master and the maintenance release branches as appropriate.
+-->
```

Comment:
Worth adding "please provide a description"? All too often we're having PRs that just say "closes X" and it takes a fair bit of reading of the original issue + comments to figure out what the change is supposed to do.

Reply:
Agreed!